adapters/between/between.go

```diff
@@ -129,14 +125,6 @@ func buildImpBanner(imp *openrtb2.Imp) error {
 // Add Between required properties to Imp object
 func addImpProps(imp *openrtb2.Imp, secure *int8, betweenExt *openrtb_ext.ExtImpBetween) {
```

Comment:
You can drop passing `betweenExt` since it is no longer used. You may also consider dropping the function call entirely, since it is now a one line function. Unless you have plans for expanding this function again in the future.